aggregator/src/aggregation/blob_data.rs

```diff
@@ -148,6 +148,21 @@ impl BlobDataConfig {
             },
         );
 
+        meta.create_gate("BlobDataConfig (boolean columns)", |meta| {
+            let is_data = meta.query_selector(config.data_selector);
+            let is_hash = meta.query_selector(config.hash_selector);
+
+            let cond = is_data.clone() + is_hash.clone() - is_data * is_hash;
```

Comment:
`is_data` and `is_hash` will not be `true` simultaneously. So the `is_data * is_hash` is redundant.

- `is_data` is true for the "chunk data" section
- `is_hash` is true for the "digest rlc" section

Reply:
Also, if you did mean to add the boolean conditions for the "metadata section" as well, then `cond` does not cover this section as `is_data` and `is_hash` are both `false` for the "metadata" section as well as the "digest bytes" section.

Reply:
The reason why I only added these boolean checks at the `is_data` condition, i.e. "chunk data" section is that:
This is the only region where `is_boundary` occurs at dynamic offsets and triggers a lookup to the keccak table. Also the `is_padding` is meaningful only for this section, as the other sections do not expect padding, i.e. we expect `is_padding == false`, but since they are never queried, it is OK to skip that check?